frontend/src/features/admin-form/create/builder-and-design/MagicFormBuilder/components/MagicFormBuilderPromptModal.tsx

```diff
@@ -138,19 +151,32 @@ const TextPromptModalBodyContent = ({
           borderRadius="4px"
           placeholder={GENERATE_FORM_PLACEHOLDER}
           {...register('prompt', {
-            required: 'Please enter a prompt',
+            required: t('promptRequiredError'),
             maxLength: {
```

Comment:
what does this do? utilise the maxLength check and update the message to include the counter
why? rely on react hook form to do the validation